htdocs/.htaccess

```diff
@@ -15,4 +15,15 @@ AddOutputFilterByType DEFLATE text/html text/plain text/xml text/css application
 	ExpiresActive on
     ExpiresDefault "access plus 1 day"
 </FilesMatch>
+
+<FilesMatch "styles\.json$">
+	ExpiresActive on
+	ExpiresDefault "access plus 1 hour"
+</FilesMatch>
+
+<FilesMatch "\.html">
+	ExpiresActive on
+	ExpiresDefault "access plus 1 hour"
+</FilesMatch>
+
```

Comment:
I don't think we want this, do we? Won't this prevent revalidation for an hour? If anything, we want this to be a few seconds, to avoid revalidation if people click through to View Source and back, but we want people to get the latest JSON as soon as it's ready. We just want them to get 304s in the meantime.